homeWork/src/db/service/EmployeeSelect.java

```diff
@@ -0,0 +1,62 @@
+package db.service;
+
+import db.ConnectorDB;
+import db.entity.Employee;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+
+public class EmployeeSelect {
+
+    public static void select() {
```

Comment:
Too many try-catch block. Quite difficult to read the code. Either use try-resource or enclose everything into the single try-catch blcok